test/e2e/specs/editor/plugins/nonce.spec.js

```diff
@@ -16,6 +16,7 @@ test.describe( 'Nonce', () => {
 		await admin.createNewPost();
 		await page.keyboard.press( 'Enter' );
 		// Wait until the network is idle.
+		// eslint-disable-next-line playwright/no-networkidle
```

Comment:
Someone could verify that these exceptions caught after the ESLint package upgrade should be refactored in the follow-up PR.